configure.py

```diff
@@ -0,0 +1,63 @@
+import os, platform, subprocess
+
+def checkCommand(shell):
```

Comment:
This should be check_command(), to meet PEP8 naming conventions.  Same for checkAll()

Reply:
fixed